packages/app/src/components/MenuBar/MenuBar.js

```diff
@@ -7,8 +7,14 @@ import UpdateButton from './UpdateButton';
 import VisualizationOptionsManager from '../VisualizationOptions/VisualizationOptionsManager';
 import * as fromActions from '../../actions';
 import './MenuBar.css';
+import { sGetCurrent } from '../../reducers/current';
+import { sGetVisualization } from '../../reducers/visualization';
+import { sGetUi } from '../../reducers/ui';
+import history from '../../history';
 
-const getOnOpen = props => id => props.onLoadVisualizaton(id);
+const getOnOpen = props => id => history.push(`/${id}`);
```

Comment:
At this point I guess this can turn into a normal callback, since we don't need props passed along.

Reply:
agreed